From: Paul Jackson <pj@sgi.com>
Subject: Source code formatting changes
References: 150185
Patch-mainline: yes

Rewrap the overly long source code lines resulting from the previous
patch's addition of the slab cache flag SLAB_MEM_SPREAD.  This patch
contains only formatting changes, and no function change.

Signed-off-by: Paul Jackson <pj@sgi.com>
Signed-off-by: Andrew Morton <akpm@osdl.org>
Acked-by: Andreas Gruenbacher <agruen@suse.de>

---

 fs/adfs/super.c       |    3 ++-
 fs/affs/super.c       |    3 ++-
 fs/befs/linuxvfs.c    |    3 ++-
 fs/bfs/inode.c        |    3 ++-
 fs/block_dev.c        |    3 ++-
 fs/cifs/cifsfs.c      |    3 ++-
 fs/dquot.c            |    3 ++-
 fs/ext2/super.c       |    3 ++-
 fs/ext3/super.c       |    3 ++-
 fs/fat/inode.c        |    3 ++-
 fs/hpfs/super.c       |    3 ++-
 fs/isofs/inode.c      |    3 ++-
 fs/jffs/inode-v23.c   |   10 ++++++----
 fs/jffs2/super.c      |    3 ++-
 fs/jfs/super.c        |    3 ++-
 fs/minix/inode.c      |    3 ++-
 fs/ncpfs/inode.c      |    3 ++-
 fs/nfs/direct.c       |    3 ++-
 fs/nfs/inode.c        |    3 ++-
 fs/ocfs2/dlm/dlmfs.c  |    3 ++-
 fs/ocfs2/super.c      |    8 +++++---
 fs/proc/inode.c       |    3 ++-
 fs/qnx4/inode.c       |    3 ++-
 fs/reiserfs/super.c   |    3 ++-
 fs/romfs/inode.c      |    3 ++-
 fs/smbfs/inode.c      |    3 ++-
 fs/udf/super.c        |    3 ++-
 fs/ufs/super.c        |    3 ++-
 net/socket.c          |    3 ++-
 net/sunrpc/rpc_pipe.c |    7 ++++---
 30 files changed, 69 insertions(+), 37 deletions(-)

diff -puN fs/adfs/super.c~cpuset-memory-spread-slab-cache-format fs/adfs/super.c
--- devel/fs/adfs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/adfs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -241,7 +241,8 @@ static int init_inodecache(void)
 {
 	adfs_inode_cachep = kmem_cache_create("adfs_inode_cache",
 					     sizeof(struct adfs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (adfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/affs/super.c~cpuset-memory-spread-slab-cache-format fs/affs/super.c
--- devel/fs/affs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/affs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -98,7 +98,8 @@ static int init_inodecache(void)
 {
 	affs_inode_cachep = kmem_cache_create("affs_inode_cache",
 					     sizeof(struct affs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (affs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/befs/linuxvfs.c~cpuset-memory-spread-slab-cache-format fs/befs/linuxvfs.c
--- devel/fs/befs/linuxvfs.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/befs/linuxvfs.c	2006-02-27 18:00:02.000000000 -0800
@@ -427,7 +427,8 @@ befs_init_inodecache(void)
 {
 	befs_inode_cachep = kmem_cache_create("befs_inode_cache",
 					      sizeof (struct befs_inode_info),
-					      0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					      0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					      init_once, NULL);
 	if (befs_inode_cachep == NULL) {
 		printk(KERN_ERR "befs_init_inodecache: "
diff -puN fs/bfs/inode.c~cpuset-memory-spread-slab-cache-format fs/bfs/inode.c
--- devel/fs/bfs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/bfs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -257,7 +257,8 @@ static int init_inodecache(void)
 {
 	bfs_inode_cachep = kmem_cache_create("bfs_inode_cache",
 					     sizeof(struct bfs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (bfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/block_dev.c~cpuset-memory-spread-slab-cache-format fs/block_dev.c
--- devel/fs/block_dev.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/block_dev.c	2006-02-27 18:00:02.000000000 -0800
@@ -319,7 +319,8 @@ void __init bdev_cache_init(void)
 {
 	int err;
 	bdev_cachep = kmem_cache_create("bdev_cache", sizeof(struct bdev_inode),
-			0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD|SLAB_PANIC,
+			0, (SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+				SLAB_MEM_SPREAD|SLAB_PANIC),
 			init_once, NULL);
 	err = register_filesystem(&bd_type);
 	if (err)
diff -puN fs/cifs/cifsfs.c~cpuset-memory-spread-slab-cache-format fs/cifs/cifsfs.c
--- devel/fs/cifs/cifsfs.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/cifs/cifsfs.c	2006-02-27 18:00:02.000000000 -0800
@@ -692,7 +692,8 @@ cifs_init_inodecache(void)
 {
 	cifs_inode_cachep = kmem_cache_create("cifs_inode_cache",
 					      sizeof (struct cifsInodeInfo),
-					      0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					      0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					      cifs_init_once, NULL);
 	if (cifs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/dquot.c~cpuset-memory-spread-slab-cache-format fs/dquot.c
--- devel/fs/dquot.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/dquot.c	2006-02-27 18:00:02.000000000 -0800
@@ -1817,7 +1817,8 @@ static int __init dquot_init(void)
 
 	dquot_cachep = kmem_cache_create("dquot", 
 			sizeof(struct dquot), sizeof(unsigned long) * 4,
-			SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD|SLAB_PANIC,
+			(SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+				SLAB_MEM_SPREAD|SLAB_PANIC),
 			NULL, NULL);
 
 	order = 0;
diff -puN fs/ext2/super.c~cpuset-memory-spread-slab-cache-format fs/ext2/super.c
--- devel/fs/ext2/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ext2/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -175,7 +175,8 @@ static int init_inodecache(void)
 {
 	ext2_inode_cachep = kmem_cache_create("ext2_inode_cache",
 					     sizeof(struct ext2_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (ext2_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/ext3/super.c~cpuset-memory-spread-slab-cache-format fs/ext3/super.c
--- devel/fs/ext3/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ext3/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -481,7 +481,8 @@ static int init_inodecache(void)
 {
 	ext3_inode_cachep = kmem_cache_create("ext3_inode_cache",
 					     sizeof(struct ext3_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (ext3_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/fat/inode.c~cpuset-memory-spread-slab-cache-format fs/fat/inode.c
--- devel/fs/fat/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/fat/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -518,7 +518,8 @@ static int __init fat_init_inodecache(vo
 {
 	fat_inode_cachep = kmem_cache_create("fat_inode_cache",
 					     sizeof(struct msdos_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (fat_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/hpfs/super.c~cpuset-memory-spread-slab-cache-format fs/hpfs/super.c
--- devel/fs/hpfs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/hpfs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -191,7 +191,8 @@ static int init_inodecache(void)
 {
 	hpfs_inode_cachep = kmem_cache_create("hpfs_inode_cache",
 					     sizeof(struct hpfs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (hpfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/isofs/inode.c~cpuset-memory-spread-slab-cache-format fs/isofs/inode.c
--- devel/fs/isofs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/isofs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -87,7 +87,8 @@ static int init_inodecache(void)
 {
 	isofs_inode_cachep = kmem_cache_create("isofs_inode_cache",
 					     sizeof(struct iso_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (isofs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/jffs2/super.c~cpuset-memory-spread-slab-cache-format fs/jffs2/super.c
--- devel/fs/jffs2/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/jffs2/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -331,7 +331,8 @@ static int __init init_jffs2_fs(void)
 
 	jffs2_inode_cachep = kmem_cache_create("jffs2_i",
 					     sizeof(struct jffs2_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     jffs2_i_init_once, NULL);
 	if (!jffs2_inode_cachep) {
 		printk(KERN_ERR "JFFS2 error: Failed to initialise inode cache\n");
diff -puN fs/jffs/inode-v23.c~cpuset-memory-spread-slab-cache-format fs/jffs/inode-v23.c
--- devel/fs/jffs/inode-v23.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/jffs/inode-v23.c	2006-02-27 18:00:02.000000000 -0800
@@ -1812,15 +1812,17 @@ init_jffs_fs(void)
 	}
 #endif
 	fm_cache = kmem_cache_create("jffs_fm", sizeof(struct jffs_fm),
-				     0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
-				     NULL, NULL);
+		       0,
+		       SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+		       NULL, NULL);
 	if (!fm_cache) {
 		return -ENOMEM;
 	}
 
 	node_cache = kmem_cache_create("jffs_node",sizeof(struct jffs_node),
-				       0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
-				       NULL, NULL);
+		       0,
+		       SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+		       NULL, NULL);
 	if (!node_cache) {
 		kmem_cache_destroy(fm_cache);
 		return -ENOMEM;
diff -puN fs/jfs/super.c~cpuset-memory-spread-slab-cache-format fs/jfs/super.c
--- devel/fs/jfs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/jfs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -634,7 +634,8 @@ static int __init init_jfs_fs(void)
 
 	jfs_inode_cachep =
 	    kmem_cache_create("jfs_ip", sizeof(struct jfs_inode_info), 0, 
-			    SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD, init_once, NULL);
+			    SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+			    init_once, NULL);
 	if (jfs_inode_cachep == NULL)
 		return -ENOMEM;
 
diff -puN fs/minix/inode.c~cpuset-memory-spread-slab-cache-format fs/minix/inode.c
--- devel/fs/minix/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/minix/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -80,7 +80,8 @@ static int init_inodecache(void)
 {
 	minix_inode_cachep = kmem_cache_create("minix_inode_cache",
 					     sizeof(struct minix_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (minix_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/ncpfs/inode.c~cpuset-memory-spread-slab-cache-format fs/ncpfs/inode.c
--- devel/fs/ncpfs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ncpfs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -72,7 +72,8 @@ static int init_inodecache(void)
 {
 	ncp_inode_cachep = kmem_cache_create("ncp_inode_cache",
 					     sizeof(struct ncp_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (ncp_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/nfs/direct.c~cpuset-memory-spread-slab-cache-format fs/nfs/direct.c
--- devel/fs/nfs/direct.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/nfs/direct.c	2006-02-27 18:00:02.000000000 -0800
@@ -898,7 +898,8 @@ int nfs_init_directcache(void)
 {
 	nfs_direct_cachep = kmem_cache_create("nfs_direct_cache",
 						sizeof(struct nfs_direct_req),
-						0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+						0, (SLAB_RECLAIM_ACCOUNT|
+							SLAB_MEM_SPREAD),
 						NULL, NULL);
 	if (nfs_direct_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/nfs/inode.c~cpuset-memory-spread-slab-cache-format fs/nfs/inode.c
--- devel/fs/nfs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/nfs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -2276,7 +2276,8 @@ static int nfs_init_inodecache(void)
 {
 	nfs_inode_cachep = kmem_cache_create("nfs_inode_cache",
 					     sizeof(struct nfs_inode),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (nfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/ocfs2/dlm/dlmfs.c~cpuset-memory-spread-slab-cache-format fs/ocfs2/dlm/dlmfs.c
--- devel/fs/ocfs2/dlm/dlmfs.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ocfs2/dlm/dlmfs.c	2006-02-27 18:00:02.000000000 -0800
@@ -596,7 +596,8 @@ static int __init init_dlmfs_fs(void)
 
 	dlmfs_inode_cache = kmem_cache_create("dlmfs_inode_cache",
 				sizeof(struct dlmfs_inode_private),
-				0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+				0, (SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+					SLAB_MEM_SPREAD),
 				dlmfs_init_once, NULL);
 	if (!dlmfs_inode_cache)
 		return -ENOMEM;
diff -puN fs/ocfs2/super.c~cpuset-memory-spread-slab-cache-format fs/ocfs2/super.c
--- devel/fs/ocfs2/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ocfs2/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -950,9 +950,11 @@ static void ocfs2_inode_init_once(void *
 static int ocfs2_initialize_mem_caches(void)
 {
 	ocfs2_inode_cachep = kmem_cache_create("ocfs2_inode_cache",
-					       sizeof(struct ocfs2_inode_info),
-					       0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
-					       ocfs2_inode_init_once, NULL);
+				       sizeof(struct ocfs2_inode_info),
+				       0,
+				       (SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
+				       ocfs2_inode_init_once, NULL);
 	if (!ocfs2_inode_cachep)
 		return -ENOMEM;
 
diff -puN fs/proc/inode.c~cpuset-memory-spread-slab-cache-format fs/proc/inode.c
--- devel/fs/proc/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/proc/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -121,7 +121,8 @@ int __init proc_init_inodecache(void)
 {
 	proc_inode_cachep = kmem_cache_create("proc_inode_cache",
 					     sizeof(struct proc_inode),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (proc_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/qnx4/inode.c~cpuset-memory-spread-slab-cache-format fs/qnx4/inode.c
--- devel/fs/qnx4/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/qnx4/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -546,7 +546,8 @@ static int init_inodecache(void)
 {
 	qnx4_inode_cachep = kmem_cache_create("qnx4_inode_cache",
 					     sizeof(struct qnx4_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (qnx4_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/reiserfs/super.c~cpuset-memory-spread-slab-cache-format fs/reiserfs/super.c
--- devel/fs/reiserfs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/reiserfs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -521,7 +521,8 @@ static int init_inodecache(void)
 	reiserfs_inode_cachep = kmem_cache_create("reiser_inode_cache",
 						  sizeof(struct
 							 reiserfs_inode_info),
-						  0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+						  0, (SLAB_RECLAIM_ACCOUNT|
+							SLAB_MEM_SPREAD),
 						  init_once, NULL);
 	if (reiserfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/romfs/inode.c~cpuset-memory-spread-slab-cache-format fs/romfs/inode.c
--- devel/fs/romfs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/romfs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -579,7 +579,8 @@ static int init_inodecache(void)
 {
 	romfs_inode_cachep = kmem_cache_create("romfs_inode_cache",
 					     sizeof(struct romfs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (romfs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/smbfs/inode.c~cpuset-memory-spread-slab-cache-format fs/smbfs/inode.c
--- devel/fs/smbfs/inode.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/smbfs/inode.c	2006-02-27 18:00:02.000000000 -0800
@@ -80,7 +80,8 @@ static int init_inodecache(void)
 {
 	smb_inode_cachep = kmem_cache_create("smb_inode_cache",
 					     sizeof(struct smb_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (smb_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/udf/super.c~cpuset-memory-spread-slab-cache-format fs/udf/super.c
--- devel/fs/udf/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/udf/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -140,7 +140,8 @@ static int init_inodecache(void)
 {
 	udf_inode_cachep = kmem_cache_create("udf_inode_cache",
 					     sizeof(struct udf_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (udf_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN fs/ufs/super.c~cpuset-memory-spread-slab-cache-format fs/ufs/super.c
--- devel/fs/ufs/super.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/fs/ufs/super.c	2006-02-27 18:00:02.000000000 -0800
@@ -1184,7 +1184,8 @@ static int init_inodecache(void)
 {
 	ufs_inode_cachep = kmem_cache_create("ufs_inode_cache",
 					     sizeof(struct ufs_inode_info),
-					     0, SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+					     0, (SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
 					     init_once, NULL);
 	if (ufs_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN net/socket.c~cpuset-memory-spread-slab-cache-format net/socket.c
--- devel/net/socket.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/net/socket.c	2006-02-27 18:00:02.000000000 -0800
@@ -319,7 +319,8 @@ static int init_inodecache(void)
 {
 	sock_inode_cachep = kmem_cache_create("sock_inode_cache",
 				sizeof(struct socket_alloc),
-				0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
+				0, (SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+					SLAB_MEM_SPREAD),
 				init_once, NULL);
 	if (sock_inode_cachep == NULL)
 		return -ENOMEM;
diff -puN net/sunrpc/rpc_pipe.c~cpuset-memory-spread-slab-cache-format net/sunrpc/rpc_pipe.c
--- devel/net/sunrpc/rpc_pipe.c~cpuset-memory-spread-slab-cache-format	2006-02-27 18:00:02.000000000 -0800
+++ devel-akpm/net/sunrpc/rpc_pipe.c	2006-02-27 18:00:02.000000000 -0800
@@ -850,9 +850,10 @@ init_once(void * foo, kmem_cache_t * cac
 int register_rpc_pipefs(void)
 {
 	rpc_inode_cachep = kmem_cache_create("rpc_inode_cache",
-                                             sizeof(struct rpc_inode),
-                                             0, SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|SLAB_MEM_SPREAD,
-                                             init_once, NULL);
+				sizeof(struct rpc_inode),
+				0, (SLAB_HWCACHE_ALIGN|SLAB_RECLAIM_ACCOUNT|
+						SLAB_MEM_SPREAD),
+				init_once, NULL);
 	if (!rpc_inode_cachep)
 		return -ENOMEM;
 	register_filesystem(&rpc_pipe_fs_type);
_
